src/test/java/org/broadinstitute/consent/http/service/DarCollectionServiceTest.java

```diff
@@ -306,17 +318,47 @@ void testCreateElectionsForDarCollection() throws Exception {
     dar.setReferenceId(UUID.randomUUID().toString());
     DarCollection collection = createMockCollections().get(0);
     collection.addDar(dar);
-    when(darCollectionServiceDAO.createElectionsForDarCollection(any(), any())).thenReturn(
+    when(darCollectionServiceDAO.createElectionsForDarByUser(any(), any())).thenReturn(
         List.of("electionId"));
     when(voteDAO.findVoteUsersByElectionReferenceIdList(any())).thenReturn(List.of(new User()));
 
     service.createElectionsForDarCollection(user, collection);
-    verify(darCollectionServiceDAO).createElectionsForDarCollection(any(), any());
+    verify(darCollectionServiceDAO, times(1)).createElectionsForDarByUser(any(), eq(dar));
+    verify(voteDAO, times(1)).findVoteUsersByElectionReferenceIdList(any());
+    verify(darCollectionDAO, times(1)).findDARCollectionByCollectionId(any());
```

Comment:
`times(1)` is redundant as this is the default for `verify()`
```suggestion
    verify(darCollectionServiceDAO).createElectionsForDarByUser(any(), eq(dar));
    verify(voteDAO).findVoteUsersByElectionReferenceIdList(any());
    verify(darCollectionDAO).findDARCollectionByCollectionId(any());
```

Also, these calls are already verified below. And, `createElectionsForDarByUser()` and `findVoteUsersByElectionReferenceIdList()` do not need to be verified at all because they are mocked, and this test uses strict stubbing.

Reply:
This was a result of a merge with your recent PR.   Bottom line is that we were both adding coverage.  Now consolidated.